cypress/support/step-definitions/textbox-steps.js

```diff
@@ -4,7 +4,7 @@ import { label } from '../../locators';
 const TEXT_ALIGN = 'text-align';
 
 Then('Textbox placeholder is set to {string}', (text) => {
-  textbox().should('have.attr', 'placeholder', text);
+  // textbox().should('have.attr', 'placeholder', text);
```

Comment:
Does this commented-out line need to remain here for future reference, or can it be deleted?

Reply:
Yes, it should be deleted :) My fault

Reply:
No worries! Thanks for removing it in fb3c1b2 :thumbsup: